docs/en/03_configuration/30_creating_an_index.md

```diff
@@ -0,0 +1,23 @@
+# Creating an index
+
+An index can essentially be considered a database that contains all of your searchable content. By default, it will store everything in a field called `Content`, which is queried to find your search results. To create an index that you can query, you can define it like so:
+
+```php
+use Page;
+use SilverStripe\FullTextSearch\Solr\SolrIndex;
+
+class MyIndex extends SolrIndex
+{
+    public function init()
+    {
+        $this->addClass(Page::class);
+        $this->addFulltextField('Title');
+    }
+}
+```
+
+This will create a new `SolrIndex` called `MyIndex`, and it will store the `Title` field on all `Pages` for searching.
+
+You can also skip listing all searchable fields, and have the index figure it out automatically via `addAllFulltextFields()`. This will add any database fields that are `instanceof DBString` to the index.
```

Comment:
One thing that might be worth adding is that `addAllFulltextFields()` has the potential of indexing sensitive content. Depending on what classes you're indexing and the type of content it contains, explicitly defining the columns to index may be preferred.